simpleSequential/test_makeCoordinateTuple.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from makeCoordinateTuple import makeCoordinateTuple
+
+class TestMakeCoordinateTuple(unittest.TestCase):
```

Comment:
I think you only need 1 test for this one. Positive or negative does not really make any different.